src/main/java/com/laytonsmith/core/functions/World.java

```diff
@@ -1801,4 +1801,54 @@ public Version since() {
 		}
 
 	}
+
+	@api
+	public static class get_temperature extends AbstractFunction {
+
+		@Override
+		public ExceptionType[] thrown() {
+			return new Exceptions.ExceptionType[]{Exceptions.ExceptionType.FormatException};
+		}
+
+		@Override
+		public boolean isRestricted() {
+			return false;
+		}
+
+		@Override
+		public Boolean runAsync() {
+			return false;
+		}
+
+		@Override
+		public Construct exec(Target target, Environment environment, Construct... args) throws ConfigRuntimeException {
+			MCPlayer player = environment.getEnv(CommandHelperEnvironment.class).GetPlayer();
+			MCWorld world = null;
+			if(player != null) { world = player.getWorld(); }
```

Comment:
She may ask you to follow the formatting guidelines and make this line 3 lines.
    if(player != null) { 
        world = player.getWorld();
    }